### DISCUSSION TITLE
How do I convert NER SpaCy format to IOB format

### DISCUSSION THREAD

**Discussion by NaziaToma** [explosion/spaCy#9005] [Help: Coding & Implementations] [answered]
I have my labeled data in this format: 

`{"classes":["HOLDING_NUMBER","ROAD_NAME_NUMBER","SUPER_SUB_AREA","SUB_AREA","AREA"],"annotations":[["Zoo Road, Bir Bikram Hemayet Uddin Sarak Section 2 Mirpur",{"entities":[[0,39,"ROAD_NAME_NUMBER"],[41,50,"SUB_AREA"],[51,57,"AREA"]]}]`

How do I convert it into IOB format in a csv file? Output should look like this-

Sentence # | Word | Pos | Tag
Sentence 1 | Zoo |  PROPN | B-ROAD_NAME_NUMBER
Sectence 1| Road | PROPN | B-ROAD_NAME_NUMBER .... continued

**Comment by polm** [comment, marked as answer]
spaCy has some tools to convert IOB data to a spaCy format, but there is no built-in functionality for converting spaCy data into IOB. 

If you want to make IOB data from spaCy data, you should be able to just load docs and do a for loop over tokens to generate it.
